FileSync/SyncDiff/Client.pm

```diff
@@ -58,6 +58,8 @@ use Scalar::Util qw(looks_like_number);
 # Debugging
 #
 
+our $DEBUG = 0;
```

Comment:
I can see rhyme and reason you may want this locally to a functional area, particularly for testing.  What I'd suggest is leave $DEBUG as is, but when the functional area gets instantiated read out $config and check if --debug or a debug config option was passed in, then set this to true depending on that check (and obviously this gives you a way to override that to a 1 regardless.